tpgtools/overrides/bigqueryreservation/samples/assignment/basic.tf.tmpl

```diff
@@ -1,5 +1,5 @@
 resource "google_bigquery_reservation" "basic" {
-  name  = "tf-test-my-reservation%{random_suffix}"
```

Comment:
Just realized we can't remove the line here. I have a PR to replace it, but then I found out the test is failing: https://github.com/GoogleCloudPlatform/magic-modules/pull/7818

I will set out some time to fix the test, but for now you should remove from this PR